tools_init/_core.sh

```diff
@@ -32,6 +32,16 @@ function build_rules_array {
   IFS="${orig_IFS}"
 }
 
+### $1 : the variable to set into the computed response (using it by reference)
```

Comment:
please change this comment : 
- $1 is not used in method
- make a desctiption of method purpose